### ISSUE TITLE
Can't turn on singleQuote

### ISSUE THREAD

**Issue by drorp24** [prettier/prettier-atom#427]
Guys,
I've been fighting this for days now, to no avail. Appreciate your help! Here are the details:

- **prettier-atom config**'s `Single Quote` is on, `ESLing Integration` is off (prettier-atom screenshot enclosed)
- **project's local .eslintrc.js** file has `extends: ['eslint:recommended', 'prettier']`, `plugins: ['react', 'prettier']` and `'prettier/prettier': 'error'`, exactly as prescribed by prettier in [Integrating with ESLint](https://prettier.io/docs/en/eslint.html)  (.eslintrc.js screenshot enclosed)
- **package.json** has `eslint-config-prettier`, `eslint-plugin-prettier`, `eslint-plugin-react` and `prettier` devDependencies defined (enclosed screenshot)
- **prettier-atom debug** clearly shows `singleQuote: true`
- at the same time (see screenshot) ESLint, on bahalf of prettier/prettier, asks to replace all single quotes with double quotes 
- removing `'prettier/prettier': 'error'` from .eslintrc.js makes all these quote errors go away. But I'm afraid I may be throwing the baby out with the bathwater.
- You may have noticed that ESLint's own "quotes" rule was overridden to "single", so ESLint too should not want quotes to be double. 
- no other .eslintrc.x file nor any .prettierrc file of any kind exist anywhere on my computer (not that it would matter, as .eslintrc.js is in the project's local root directory).

There's no rule anywhere to suggest that quotes should be double. Yet, prettier/prettier errs nearly every line in the codebase, making it really really hard to be productive.

Appreciate any help, thanks.

![image](https://user-images.githubusercontent.com/594441/41619079-9d2a95dc-740d-11e8-8b00-4a2d4652948a.png)
![image](https://user-images.githubusercontent.com/594441/41619219-15cd16f4-740e-11e8-9023-25a46acdcf2b.png)
![image](https://user-images.githubusercontent.com/594441/41619855-648c38c2-7410-11e8-9653-e03e85d8dbf9.png)
![image](https://user-images.githubusercontent.com/594441/41620085-49ce433a-7411-11e8-87f1-7ce507c77403.png)

**Comment by robwise** [comment]
FYI `eslint-config-prettier` turns off ESLint's control of anything that Prettier handles, which includes quotes, but you've manually re-specified this rule. There's no need to do that although you probably did that just to troubleshoot. However, as you can see in your screenshots, the error is coming via the `prettier/prettier` rule and not the quote rule being violated. 

Since you have no `.prettierrc`, prettier-atom is defaulting back to your Atom settings for prettier, which is why you actually get single quotes as desired.

However, `eslint-plugin-prettier` has no way of knowing you configured prettier-atom. This plugin just looks for a `prettierrc`, can't find it, and defaults back to prettier's default settings of double quotes. This is where the double quotes comes from—it's a default prettier setting when there is no prettierrc specified. Therefore even though prettier-atom formats correctly, the ESLint plugin thinks you wanted double quotes and shows an error.

You should be setting your Prettier settings via an `.prettierrc` file as a general practice. In an upcoming version of prettier-atom we will be eliminating atom-specific prettier settings entirely and instead requiring using a .prettierrc. This is for a multitude of reasons, one of which is to reduce this type of confusion.

I assume that adding such a file and specifying single quotes there should solve your problem.

**Comment by drorp24** [comment]
Thank you so much. Not only did the .prettierrc solve the problem like you said but I got to understand what has happened here, towards hopefully  a more educated use of both tools going forward.

I suspected there was probably no reason for me to add the single quote rule to ESLint as it had been turned off on anything Prettier handles by `eslint-config-prettier`, but I wasn't 100% sure.

I do admit that I got lost between the .eslintrc, the .prettierrc and the prettier-atom configurations. I was hoping that someone would help me understand how they all relate. Thanks for making sense of everything.
